lib/src/main/java/uk/gov/di/ipv/core/library/domain/IpvJourneyTypes.java

```diff
@@ -13,4 +13,8 @@ public enum IpvJourneyTypes {
     public String getValue() {
         return value;
     }
+
+    public static IpvJourneyTypes get(String journeyType) {
+        return IpvJourneyTypes.valueOf(journeyType.toUpperCase().replace("-", "_"));
+    }
```

Comment:
I'm wondering if rather than adding this method, we just set the value of the parameter to the expected value (`IPV_CORE_MAIN_JOURNEY` for example), then we can just call `IpvJourneyTypes.valueOf(journeyType)`